src/clients/mobiles/remote/playback/playback.tsx

```diff
@@ -35,25 +39,32 @@ export class Playback extends React.Component<PlaybackPops, State>  {
                 timeAsPercentage: props.currentTimeAsPercentage
             })
         }
+        if (this.state.dominantBackgroundColor! = props.dominantBackgroundColor) {
+            this.setState({
+                dominantBackgroundColor: props.dominantBackgroundColor
+            })
+
+            changeCSSVar('progress-main-color', this.state.dominantBackgroundColor);
+            changeCSSVar('slider-main-color', this.state.dominantBackgroundColor);
+        }
     }
 
-    public _onTimeChange = () => {
-        const timeAsPercentage = parseInt(this.refs.time.value)
-        const time = timeAsPercentage* this.props.duration /100
+    public _onTimeChange = (value: number) => {
+        const time = (value * this.props.duration) / 100
         this.setState({
-            timeAsPercentage: timeAsPercentage
+            timeAsPercentage: value
         })
         if (this.props.onTimeChange)
             this.props.onTimeChange(time)
     }
+
     public render() {
         return (
             <div className={styles.timeSlider}>
-                <input ref="time" type="range" min="1" max="100"
-                    value={this.state.timeAsPercentage.toString()}
+                <Slider value={this.state.timeAsPercentage}
```

Comment:
You can get rid of the manually binding if you convert  ontimechanged to pure puction ( not an array function )

Reply:
Actually ```() => this._onTimeChange()``` doesn't work with the way sliders are handled in [React Toolbox](http://react-toolbox.com/#/components/slider)
If you see how to enable it let me know !